ewidth.py

```diff
@@ -1,182 +1,221 @@
-
+ 
 # Jean-paul Ventura
 # January 12, 2017
 # Modified: February 20th, 2016
```

Comment:
It's not clear what the significance of these dates are and I'm not sure that you need them because GitHub tracks when you change things.